.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
The `eslint-config-kentcdodds` update changed some rules including removing the quotes and arrow parenthesis rules.